spec/helpers/metadata.ts

```diff
@@ -22,4 +23,5 @@ class MetadataHelper {
     }
 }
 
-module.exports = new MetadataHelper();
+export const metadata = new MetadataHelper();
+export default metadata;
```

Comment:
Why do we need both? `import metadata, {metadata}`?

Reply:
I'm not sure if we no longer need that. At first, my plan was to only port a subset of tests and I left both exports for compatibility with `commonjs` imports. It turned out faster and I ported all tests, so I'll remove these double exports.